src/prefect/tasks/github/comments.py

```diff
@@ -0,0 +1,87 @@
+import json
+from typing import Any
+
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+
+
+class CreateIssueComment(Task):
+    """
+    Task for creating a comment on the specified GitHub issue using the v3 version of the
+    GitHub REST API.
+
+    Args:
+        - repo (str, optional): the name of the repository to create the comment in; must be
+            provided in the form `organization/repo_name`; can also be provided to the `run`
+            method
+        - issue_number (int, optional): the ID of the issue to create the comment on.
+        - body (str, optional): the body of the comment; can also be provided to the `run` method
+        - **kwargs (Any, optional): additional keyword arguments to pass to the standard Task
+            init method
+
+    References:
+        - See: [Create an issue
+            comment](https://developer.github.com/v3/issues/comments/#create-an-issue-comment)
```

Comment:
@jcrist Adding a referecne section is more complicated than I expected. Currently, a list item has to follow the following format (otherwise `format_lists` in `generate_doc.py` would ignore it):

```
- xxx: yyy
```

Reply:
To allow a format like:

```
- xxx
```

We'd need to fix `format_lists`.

Reply:
The current code is rendered like this:

<img width="362" alt="Screen Shot 2020-09-09 at 22 12 18" src="https://user-images.githubusercontent.com/17039389/92602838-a0f60180-f2e9-11ea-9dbf-c39b785c9371.png">

https://5f58cff0ed4e1e00076692e2--prefect-docs.netlify.app/api/latest/tasks/github.html#createissuecomment

Reply:
Hmmm, that's unfortunate. The fix you've done here looks good. Thanks!